lib/scrabble.rb

```diff
@@ -0,0 +1,64 @@
+# require_relative '../lib/scrabble_player'
+
+module Scrabble
+
+  class Scoring
+
+    attr_reader :SCOREBOARD
+
+    # constant that is a hash of the point values of the letters
+    SCOREBOARD = {
+      %w[a e i o u l n r s t]=>1,
+      %w[d g]=> 2, %w[b c m p]=>3,
+      %w[f h v w y]=> 4,
+      %w[k]=> 5,
+      %w[j x]=> 8,
+      %w[q z]=> 10
+    }
```

Comment:
Good to use a hash, but it might have been easier in your `score` method if you had done:
```
SCOREBOARD = {
  'a' => 1,
  'e' => 1,
  'f' => 4,
  ...
}
```